bank/services/stoneBaas/validations.go

```diff
@@ -0,0 +1,31 @@
+package stoneBaas
+
+import (
+	"github.com/mundipagg/boleto-api/models"
+	"github.com/mundipagg/boleto-api/validations"
+)
+
+func validateBuyerDocumentNumber(boletoRequest interface{}) error {
+	switch t := boletoRequest.(type) {
+	case *models.BoletoRequest:
+		if t.Buyer.Document.Number == "" {
```

Comment:
Para simplificar a leitura talavez 😀 🎸 
```suggestion
		if t.Buyer.Document.Number == "" {
			return nil
		}
		
		return validations.ValidateBuyerDocumentNumber(t)
```